### ISSUE TITLE
LTX-Video support

### ISSUE THREAD

**Issue by yoavhacohen** [junhahyung/STGuidance#1]
Any plans to support LTX-Video?
https://github.com/Lightricks/LTX-Video

**Comment by CCpt5** [comment]
Speedy developer logtd has added it to his LTVTricks repo: https://github.com/logtd/ComfyUI-LTXTricks?tab=readme-ov-file#enhance

**Comment by kinam0252** [comment]
Thanks for the suggestion! We're currently working on it—stay tuned for updates. :)

**Comment by kinam0252** [comment]
> Any plans to support LTX-Video? https://github.com/Lightricks/LTX-Video

We recently uploaded the code with STG applied to LTX-Video! 

We found that STG-A (scale 1.0, block index 19) with rescaling improves the quality of video (720x1280, 121 frames), but you might discover even better results by testing it with different configurations.

https://github.com/user-attachments/assets/4cc3b06c-1888-4eb2-84fb-a851ddf3f253

Thank you for creating such a great video model! ☺️
